### DISCUSSION TITLE
Inserting an actual tab instead of opening the auto completion menu.

### DISCUSSION THREAD

**Discussion by riceset** [hrsh7th/nvim-cmp#914] [Q&A]
### FAQ

- [X] I have checked the [FAQ](https://github.com/hrsh7th/nvim-cmp/blob/15f08a8faa22d52480cdcb9ef9ca698120f04363/doc/cmp.txt#L616) and it didn't resolve my problem.

### Issues

- [X] I have checked [existing issues](https://github.com/hrsh7th/nvim-cmp/issues) and there are no open or closed issues with the same problem.

### Neovim Version

0.7.0

### Minimal reproducible full config

```lua
lvim.log.level = "warn"
lvim.format_on_save = false
vim.opt.cursorline = false
vim.opt.smartindent = false
vim.opt.expandtab = false
vim.opt.shiftwidth = 4
vim.cmd("set noruler")
vim.opt.tabstop = 4
lvim.format_on_save = false
vim.cmd("set path+=**")
vim.cmd("let &fcs='eob: '")
vim.cmd("nnoremap <esc> :noh<return><esc>")
lvim.leader = "space"
lvim.keys.normal_mode["<C-s>"] = ":w<cr>"
vim.cmd("autocmd CursorHold * :echo")
vim.opt.showtabline = 1
lvim.builtin.treesitter.indent = { enable = true, disable = { "c" } }
lvim.colorscheme = "sonokai"
vim.cmd("let g:sonokai_style = 'atlantis'")
vim.cmd("let g:sonokai_enable_italic=1")
vim.cmd("let g:sonokai_disable_italic_comment=1")
vim.cmd("let g:sonokai_transparent_background = 1")
vim.cmd("let g:sonokai_sign_column_background = 'none'")
lvim.builtin.which_key.mappings["m"] = { "<cmd>BufferLineCycleNext<CR>", "Go to the next buffer" }
lvim.builtin.which_key.mappings["n"] = { "<cmd>BufferLineCyclePrev<CR>", "Go to the next buffer" }
lvim.builtin.alpha.active = true
lvim.builtin.alpha.mode = "dashboard"
lvim.builtin.notify.active = true
lvim.builtin.terminal.active = true
lvim.builtin.nvimtree.setup.view.side = "left"
lvim.builtin.nvimtree.show_icons.git = 0
lvim.builtin.treesitter.ensure_installed = {
  "bash",
  "c",
  "go",
}
lvim.builtin.treesitter.ignore_install = { "haskell" }
lvim.builtin.treesitter.highlight.enabled = true
lvim.plugins = {
  {
    "42Paris/42header",
    config = function()
      vim.cmd("let g:user42 = 'tkomeno'")
      vim.cmd("let g:mail42 = 'tkomeno@student.42tokyo.jp'")
    end
  },
  { "riceset/sonokai" },
  {
    "andweeb/presence.nvim",
    require("presence"):setup({
      auto_update        = true,
      neovim_image_text  = "Neovim",
      main_image         = "file",
      client_id          = "793271441293967371",
      log_level          = nil,
      debounce_timeout   = 10,
      enable_line_number = false,
      blacklist          = {},
      buttons            = true,
      file_assets        = {},
      editing_text        = "Editing %s",
      file_explorer_text  = "Browsing %s",
      git_commit_text     = "Committing changes",
      plugin_manager_text = "Managing plugins",
      reading_text        = "Reading %s",
      workspace_text      = "Working on %s",
      line_number_text    = "Line %s out of %s",
    })
  },
}
```

### Description

How can I insert an actual tab instead of opening the autocomplete menu? (On my school, I am required to insert a tab after every type such as int, char, double, etc)

I don't want to disable the menu. I would like to navigate through it only using the arrow keys.
(Which I have set up as caps lock + Vim keys with Karabiner.)

### Steps to reproduce

Open a C file and type `int` for instance.

### Expected behavior

I wanted an actual tab character to be placed.
![スクリーンショット 2022-04-17 午後8 09 44](https://user-images.githubusercontent.com/48802655/163735556-be557e2a-1ade-477a-9a44-cf82657c249a.png)

### Actual behavior

It navigates through the menu.
![スクリーンショット 2022-04-17 午後8 08 15](https://user-images.githubusercontent.com/48802655/163735510-33ec71ba-4c3b-4cec-b09b-de3b8c5cf6ee.png)

### Additional context

I am using [LunarVim](https://github.com/LunarVim/LunarVim).

**Comment by hrsh7th** [comment]
It's LunarVim 's default configuration.
https://github.com/LunarVim/LunarVim/blob/rolling/lua/lvim/core/cmp.lua#L257

**Reply by luiz00martins** [reply]
Yeah. Remove the `[<Tab>]` mapping and you're golden.
